libs/core/langchain_core/tracers/core.py

```diff
@@ -50,6 +50,8 @@ class _TracerCore(ABC):
     This class provides common methods, and reusable methods for tracers.
     """
 
+    log_missing_parent: bool = True
```

Comment:
Why class level attribute?

Reply:
It's more a function of the type of tracer ya?

Could also add to `__init__` and just set in the `with_(async)_listeners()` call

Reply:
i usually rely only on __init__

Reply:
Anyway up to you works either way!